public/components/GeoPlot.vue

```diff
@@ -511,11 +642,50 @@ export default Vue.extend({
       this.$bvToast.hide(this.toastId);
       window.removeEventListener("mousemove", this.fadeToast); // remove event listener because toast is now faded
     },
+    bucketsToQuads(): Quad[] {
+      const maxVal = this.maxBucketCount;
+      const minVal = this.minBucketCount;
+      const d = (maxVal - minVal) / BLUE_PALETTE.length;
+      const domain = BLUE_PALETTE.map((val, index) => minVal + d * (index + 1));
+      const scaleColors = scaleThreshold()
+        .range(BLUE_PALETTE as any)
+        .domain(domain);
+      const singleBuffer = [];
```

Comment:
I remember you explaining that WebGL uses a buffer for rendering. Do you mind adding a comment explaining this? This constant is created purely for WebGL, and it's worth mentioning.